components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/InboundAuthenticationRequestConfig.java

```diff
@@ -158,4 +160,16 @@ public String getFriendlyName() {
     public void setFriendlyName(String friendlyName) {
         this.friendlyName = friendlyName;
     }
+
+    private Property[] sortPropertiesByDisplayOrder ( Set<Property> propertySet) {
+
+        List<Property> list = new ArrayList(propertySet);
```

Comment:
Array -> List -> Set -> List -> Array can be simplified as Array -> List -> Array.